lib/v2/huggingface/daily-papers.js

```diff
@@ -0,0 +1,24 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { body: response } = await got('https://huggingface.co/papers');
+    const $ = cheerio.load(response);
+    const papersJson = $('main > div[class="SVELTE_HYDRATER contents"]')[0].attribs['data-props'];
+    const papers = JSON.parse(papersJson);
```

Comment:
From my testing, the second match contains the required json
![image](https://github.com/DIYgod/RSSHub/assets/11386903/d0c5bf23-4249-4adf-b7cb-d7129e733e83)

Maybe it's better to use to tighter match:

```suggestion
    const papers = $('main > div[data-target="DailyPapers"]').data('props');
```

Reply:
It seems that Huggingface adds a new banner just after my commit. So the automatic fetch by github-action works fine. I will update how we match the element.